src/_guides/whats-new.md

```diff
@@ -17,6 +17,80 @@ and follow the [Dart blog][].
 [dart-announce]: https://groups.google.com/a/dartlang.org/d/forum/announce
 [Dart blog]: https://medium.com/dartlang
 
+## May 19, 2021: 2.13 release
+
+This section lists notable changes made from March 4, 2021,
+through May 19, 2021.
+For details about the 2.13 release, see [Announcing Dart 2.13][].
+
+[Announcing Dart 2.13]: https://medium.com/dartlang/announcing-dart-2-13-c6d547b57067
+
+### Docs updated or added to dart.dev
+{: .no_toc}
+
+In addition to bug fixes and incremental improvements,
+we made the following changes to this site:
+
+* Updated the [typedef section][] of the language tour to reflect
+  non-function type aliases,
+  which were introduced in Dart 2.13.
+* Published or updated documentation related to the command line and servers:
+  * [Using Google Cloud][] describes Google Cloud products that
+    Dart servers can use,
+    often with the help of pre-packaged Docker images.
+  * The [HTTP server tutorial][],
+    which featured the discontinued `http_server` package,
+    has been temporarily replaced by
+    an "under construction" page that
+    links to helpful documentation and samples.
+  * The [command-line tutorial][] has been completely updated.
+* Published some other new pages:
+  * [Numbers in Dart][] has
+    details about differences between native and web number implementations.
+  * [Using Google APIs][] points to resources to
+    help you use Firebase and Google client APIs from a Dart app.
+  * [Writing package pages][] gives tips for
+    writing a package README that works well on pub.dev.
+  * [Fixing type promotion failures][]
+    has information to help you understand
+    why type promotion failures occur, and give tips on how to fix them.
```

Comment:
Perhaps `qives` or `provides` instead of `give`?

```suggestion
    why type promotion failures occur, and gives tips on how to fix them.
```

```suggestion
    why type promotion failures occur, and provides tips on how to fix them.
```